### PR TITLE
R1.5 benchmark

### PR DESCRIPTION
**Pull request by richardwxn**
cherry-picked the changes related to benchmark job itself.

Can you also cherrypick the changes you need for the dashboard part in another PR? @carolynhu

### CODE REVIEW

**Comment by richardwxn** [comment]
@carolynhu also you may need to also add upgrade test for 1.5 and also cherry-pick those upgrade related changes.

File: perf/benchmark/run_benchmark_job.sh [hunk 1]
```diff
@@ -260,36 +295,7 @@ for f in "${CONFIG_DIR}"/*; do
...
-
-CONFIG_DIR="${WD}/configs/linkerd"
-
-for f in "${CONFIG_DIR}"/*; do
-    get_benchmark_data "${f}"
-done
+#echo "collect flame graph ..."
+#collect_flame_graph
 
-echo "perf benchmark test for linkerd is done."
```
Comment (carolynhu) [inline perf/benchmark/run_benchmark_job.sh]: I think comment out linkerd part is fine.

**Comment by carolynhu** [comment]
> @carolynhu also you may need to also add upgrade test for 1.5 and also cherry-pick those upgrade related changes.

yes, will do.

File: perf/benchmark/run_benchmark_job.sh (hunk 1 above)
Comment (richardwxn) [reply perf/benchmark/run_benchmark_job.sh]: we can add it back later, next step would be adding back CPU/mem after master and 1.5 greens

**Pull request merged**